features/ALB012_Alignment-vertical-segment-radius-of-curvature.feature

```diff
@@ -0,0 +1,20 @@
+@implementer-agreement
+@ALB
+@version1
+@E00020
+Feature: ALB012 - Alignment vertical segment radius of curvature
+  The rule verifies the 'RadiusOfCurvature' design parameter for vertical alignment segments.
+
+  Background:
+    Given A model with Schema "IFC4.3"
+    Given An IfcAlignmentVertical
+    Given A relationship IfcRelNests from IfcAlignmentVertical to IfcAlignmentSegment and following that
+    Given Its attribute DesignParameters
+
+  Scenario: Validating the absence of curvature radius for constant gradient vertical segment
+    Given PredefinedType != 'ARC' or 'PARABOLICARC'
+    Then The value of attribute RadiusOfCurvature must be empty
+
+  Scenario: Validating the radius of curvature for parabolic segments
+    Given PredefinedType = 'PARABOLICARC'
+    Then RadiusOfCurvature value must be equal to the expression: HorizontalLength / ( EndGradient - StartGradient )
```

Comment:
Not really consistent wording with:

> The value of attribute RadiusOfCurvature must be empty

```suggestion
    Then The value of attribute RadiusOfCurvature must be equal to the expression: HorizontalLength / ( EndGradient - StartGradient )
```

Reply:
It looks like this was done so that the implementation could stay separate from what was existing previously.  I have started work on combining the new 'expression' capabilities to the existing implementation.  ATM the new code path is working but at expense of now failing when processing the original implementations.